app/views/candidate_interface/references/review/show.html.erb

```diff
@@ -6,13 +6,23 @@
 </h1>
 
 <%= render CandidateInterface::AddReferenceComponent.new(@application_form) %>
+<% if @too_many_references %>
+  <div class="govuk-error-summary" tabindex="-1" role="alert" data-module="govuk-error-summary" aria-labelledby="error-summary-title">
+    <h2 id="error-summary-title" class="govuk-error-summary__title">
+      There is a problem
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <li><%= link_to t('application_form.references.review.more_than_two'), '#references_given' %></li>
+      </ul>
+    </div>
+  </div>
+<% end %>
```

Comment:
* The error summary should appear before the page title.
* The page title should be prefixed with `Error: `